src/features/user-profile/pages/UserProfilePage.tsx

```diff
@@ -33,13 +33,48 @@ import { ProfileFormData, UpdateUserProfileForm } from '../components/UpdateUser
 import { Trans } from 'react-i18next';
 import { LoadingButton } from 'common/components/LoadingButton';
 import { Constants } from 'utils/constants';
-import { faContactCard, faUnlockKeyhole } from '@fortawesome/free-solid-svg-icons';
+import { faCamera, faContactCard, faUnlockKeyhole } from '@fortawesome/free-solid-svg-icons';
 import { useModal } from 'react-modal-hook';
+import { LoadingSpinner } from 'common/components/LoadingSpinner';
 
 type RouteParams = {
   id: string;
 };
 
+const UserProfilePictureContainer = styled.div`
+  position: relative;
+  display: inline-block;
+
+  svg {
+    position: absolute;
+    top: 40%;
+    left: 42.5%;
```

Comment:
in order to center something absolutely in a way that isn't size dependent, you want to use a transform. This CSS for example centers horizontally:

```css
top: 50%;
transform: translateY(-50%);
```

You can do the same on the X axis and `left` to get it fully centered.

Reply:
When I do the above (while setting the X axis in the same way), the icon does not look centered to the user. Here's what I mean:

![if top and transform Y is set to 50 and -50%](https://user-images.githubusercontent.com/2876874/195458504-0aa98ef0-f410-463e-93cc-d44d58a53023.png)

Doing this for the X axis is fine. However, in my opinion, using 42% and -42% for the Y axis is more in line with what people would expect. Ex:

![if left and transform X is set to 42 and -42%](https://user-images.githubusercontent.com/2876874/195458905-42ac048c-acc9-49fe-b08e-611ec6ac2861.png)

Reply:
`transform` can only have one setting, are you writing multiple `transform` statements? that will not work, only the last one will take effect. Can I see the CSS you wrote? @joshwingreene 

We don't want absolute numbers like that 42% because it makes changing the size of the icon in the future a 2 step process (with an annoying second step), instead of a 1 step process.

Reply:
Here is what your CSS should look like @joshwingreene 

```css
top: 50%;
left: 50%;
transform: translateX(-50%) translateY(-50%);
```

Reply:
I see what you mean. Yeah. I was accidentally setting it twice. And, I just tested what you posted. Looks good. Thanks for the clarification.